src/Parallel/Info.hpp

```diff
@@ -62,7 +105,15 @@ int procs_on_node(const int node_index,
   return distributed_object.procs_on_node(node_index);
 }
 
+template <typename DistribObject>
+size_t procs_on_node(const size_t node_index,
```

Comment:
I understand these ones these don't need a different name because the signature differs, but I think it's worth considering having _all_ the `size_t` functions have the suffix, just for consistency.  Otherwise people need to remember which specific ones need it and which don't